src/components/header/Folder/Folder.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { getFolder } from '@/pages/api/api';

```

Comment:
경로 문제로 인한 타입 에러 해결 필요합니다~!
```suggestion
import { getFolder } from '@/pages/api/hello';
```